x509/src/fmc_alias_cert.rs

```diff
@@ -17,19 +17,26 @@ include!(concat!(env!("OUT_DIR"), "/fmc_alias_cert_tbs.rs"));
 
 #[cfg(all(test, target_os = "linux"))]
 mod tests {
+
     use openssl::ecdsa::EcdsaSig;
```

Comment:
Since the keyusage extension is used in idevid and ldevid can we update those tests to make sure critical bit is set

Reply:
Done